.styleci.yml

```diff
@@ -0,0 +1,71 @@
+preset: psr2
```

Comment:
This is rather strange list of fixers. Can we have `recommended` or `laravel` preset, since this list of fixers is not equal to actual [PSR2 preset](https://docs.styleci.io/presets#psr2)?

Reply:
I actually went for the PSR2 preset as we are quite harsh on it. I would say it works pretty well but you are right that it is not fully PSR2. I guess we can go for `recommended` so I will just remove already used things in `recommended` and eventually add some useful stuff from PSR2.

Reply:
Updated.